checkov/sca_package/runner.py

```diff
@@ -131,12 +132,14 @@ def parse_vulns_to_records(
         vulnerabilities: list[dict[str, Any]],
         packages: list[dict[str, Any]],
         license_statuses: list[_LicenseStatus],
+        image_details: ImageDetails | None = ImageDetails
```

Comment:
do we actually need to add `image_details` to the `sca_package` runner, it will always be `None`

Reply:
We do because this method is called by https://github.com/bridgecrewio/checkov/blob/cyclonedx_for_images/checkov/sca_image/runner.py#L234

Reply:
ah missed that, they share the same parent class.